app/style/Header.less

```diff
@@ -1,123 +1,120 @@
-@import (reference) "./main.less";

```

Comment:
Can you revert this file, please ? (encoding issue I suppose)

Reply:
"./main.less" is still there.  this is just a strange bug in git thinking I did more than I did in this file. You'll see it in the green below

Reply:
Yes I saw that^^ that's about your encoding I think